docs/api-interface.md

```diff
@@ -178,11 +178,37 @@ List all groups in an assessment
 
 **Endpoint:** `GET /courses/{course name}/assessments/{assessment name}/groups`
 
-**Parameters:** [none]
+**Parameters:**
+
+
+`show_members`
+
+| key          |          | type    | description                                          |
+| ------------ |----------|---------|------------------------------------------------------|
+| show_members | optional | boolean | whether to retrieve the members of each group or not |
+
+---
 
 **Response:**
 
 A JSON object containing the group_size, a list of groups, and the assessment containing the groups.
+If `show_members` is set to true, a list of `assessment_user_datum` objects will be retrieved for each group as well.
+
+
+#### show
+
+Show the details of a group, its assignment, and its members
```

Comment:
Actually, right now it does not show the assignment in this endpoint. I will update the response.

Reply:
I have decided to only update the docs instead of adding the assessment directly to the group response, because that is not RESTful. Adding it to the Assessment User Datum object is RESTful but will make the response unnecessarily large. If the need arises to return the assessment with the group in the future it can be added.